doc/source/conf.py

```diff
@@ -15,9 +15,16 @@
 
 import sys
 import time
-from importlib.metadata import version
 from pathlib import Path
 
+# Attempt to import the version dynamically from GitHub tag.
```

Comment:
Try `try-catch` was needed to ensure that when no version info was found, the doc can be still built.